packages/app/src/__deprecated__/renderer/components/core/tooltip/element-with-tooltip.component.test.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Mudita sp. z o.o. All rights reserved.
+ * For licensing, see https://github.com/mudita/mudita-center/blob/master/LICENSE.md
+ */
+
+import React, { ComponentProps } from "react"
+import { fireEvent } from "@testing-library/react"
+import { renderWithThemeAndIntl } from "App/__deprecated__/renderer/utils/render-with-theme-and-intl"
+import ElementWithTooltip from "App/__deprecated__/renderer/components/core/tooltip/element-with-tooltip.component"
+
+type Props = ComponentProps<typeof ElementWithTooltip>
+
+const defaultProps: Props = {
+  Element: <div data-testid="element-div">Element text</div>,
+  children: <div data-test-id="tooltip">Tooltip text</div>,
+}
+
+const renderer = (extraProps?: Partial<Props>) => {
+  const props = {
+    ...defaultProps,
+    ...extraProps,
+  }
+  return renderWithThemeAndIntl(<ElementWithTooltip {...props} />)
+}
+
+describe("ElementWithTooltip component", () => {
+  test("shows tooltip on hover when no need to ellipsis text", () => {
```

Comment:
It would be good to add a test that checks if `tooltip` is not shown by default (that is - when `ElementWithTooltip` is not hovered).

Reply:
Added in [30aa052](https://github.com/mudita/mudita-center/pull/1030/commits/30aa0526662289c0a91e42b79d17d013c80a0236)